ui/src/app/components/sharded-dataflow-migration-details-form/sharded-dataflow-migration-details-form.component.ts

```diff
@@ -46,7 +51,10 @@ export class ShardedDataflowMigrationDetailsFormComponent implements OnInit {
   testingSourceConnection: boolean = false
   creatingSourceConnection: boolean = false
   creatingTargetConnection: boolean = false
+  verifyingJson: boolean = false
```

Comment:
This seems like a very long way to show a spinner while we verify, isn't there a neater way to do this? @shreyakhajanchi

Reply:
this is the way it's usually done: https://stackoverflow.com/a/51000288 . Also, `verifyingJson` is just being used to control when to display the spinner